### PR TITLE
GE Layout changes

### LINKED ISSUE #1838: Update styling on GE pages
Grey box for intro text, padding / spacing changes, etc

https://app.zeplin.io/project/5acbd659dc553ec84805afd4/screen/5b758b93a9329e48562a94d7

### PR DESCRIPTION
**Pull request by brdunfield**
Closes #1838 

Does not change any fonts / colours / button styling, just layout on the GE pages.

### CODE REVIEW

**Comment by brdunfield** [comment]
[Implement UI changes to match new VAC site](https://trello.com/c/WzhdqieD/196-implement-ui-changes-to-match-new-vac-site)

File: components/radio.js [hunk 1]
```diff
@@ -33,10 +33,10 @@ const labelStyle = css({
   fontFamily: globalTheme.fontFamily,
   fontWeight: 400,
   textTransform: "none",
-  fontSize: "16px",
+  fontSize: "24px",
```
Comment (SupeDeDupe) [inline components/radio.js:36]: I think this needs to be done for the checkbox labels in the needs section as well. The needs font size in the mobile version can also be changed from 14px to 16px.

**Review by SupeDeDupe — COMMENTED** [review]
The spacing and padding look really good!

Not sure if the alignment and positioning of the buttons on mobile are intended to be taken care of here or in a later PR, but I think it would make sense to do it here since the positions were changed in non-mobile:

screen shot 2019-02-18 at 4 55 22 pm

-->

screen shot 2019-02-18 at 4 55 30 pm

The question tooltip font should be 24px and 14px in mobile; right now it's 18px in both. Tooltip:
screen shot 2019-02-18 at 5 00 14 pm

File: components/guided_experience.js [hunk 2]
```diff
@@ -142,30 +169,30 @@ export class GuidedExperience extends Component {
...
-                  <Body>
+            <Grid item xs={12}>
+              <Header size="lg" headingLevel="h1">
+                {t("ge.Find benefits and services")}
+              </Header>
+              {id === "patronType" ? (
+                <React.Fragment>
+                  <Body className={greyBox}>
                     <p>{t("ge.intro_text_p1")}</p>
                     <p>{t("ge.intro_text_p2")}</p>
```
Comment (SupeDeDupe) [inline components/guided_experience.js:194]: These p font sizes can probably be set to 20px as per the design.

**Comment by brdunfield** [comment]
I was planning to do the buttons in a separate PR today. It makes sense to do all the button work together and I'm not sure that work should be encapsulated in this PR. Thoughts? @SupeDeDupe

**Comment by SupeDeDupe** [comment]
Yeah, I think it makes sense to do it all together, but I guess I'm just a bit confused why the button positioning is being changed for non-mobile in this PR. I think it's fine to do the positioning for the buttons on mobile in a different PR; just wanted to make sure it's not being forgotten about.

**Comment by brdunfield** [comment]
It was changed for both mobile and not mobile - I changed it to match the design for non-mobile, and tried to get mobile to a state where it looks good enough to merge the PR. It's not being forgotten about, it was just my attempt to split the work between the issues, and leave the app looking decent after this PR.

File: components/checkbox.js [hunk 3]
```diff
@@ -40,12 +40,10 @@ const StyledLabel = styled("span")({
   fontFamily: globalTheme.fontFamily,
   fontWeight: 400,
   textTransform: "none",
-  fontSize: "14px",
-  lineHeight: "16px",
+  fontSize: "24px",
```
Comment (SupeDeDupe) [inline components/checkbox.js:44]: Still need the mobile font size to be 16px 🙈

**Review by SupeDeDupe — APPROVED** [review]
Awesome! Added one more comment. 🦑👯

**Pull request merged**